### DISCUSSION TITLE
Keyring generation with proxy credentials

### DISCUSSION THREAD

**Discussion by lapecoravolante** [microsoft/vscode-discussions#686] [Extension Development QnA]
Hello everybody,
I don't know if this is the correct place to ask but I have a quetion:
is it possible to generate the keyring file with proxy credentials under ~/.local/share/keyrings?
This would be helpfull to automate the installation with tools like Ansible and put the proxy credential during the installation, without any need for manually typing.
The problem is that vscode store the secret in a sort of hashed string and I don't know which cipher suite is used (sha, md5, ...?) nor the string format ("user:password"? "user/password"?)

**Comment by isidorn** [comment]
This community is for VS Code extension authors. Thus closing, and I suggest to ask this on StackOverflow.
